src/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -38,7 +39,7 @@ protected void launchInNewThread(final Component parent) {
     ServerGame game = null;
     try {
       m_gameData.doPreGameStartDataModifications(m_playerListing);
-      final IServerMessenger messenger = new DummyMessenger();
+      final IServerMessenger messenger = mock(IServerMessenger.class);
```

Comment:
To actually remove the DummyMessenger here, you'll need to remove all dependencies on the `IServerMessenger messenger` object in this method.